vault/resource_aws_secret_backend_static_role_test.go

```diff
@@ -0,0 +1,50 @@
+package vault
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+	"testing"
+)
+
+func TestAccAWSSecretBackendStaticRole(t *testing.T) {
+	mount := acctest.RandomWithPrefix("tf-aws-static")
+	_, _ = testutil.GetTestAWSCreds(t)
+	username := testutil.SkipTestEnvUnset(t, "AWS_STATIC_USER")[0]
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			//SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion114)
+		},
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
```

Comment:
Can we add an import step? Example: https://github.com/hashicorp/terraform-provider-vault/blob/abd3bce786b4a069360495265ab4ae03a25c1051/vault/resource_aws_secret_backend_test.go#L42